### PR TITLE
Dog194 nether wart

### PR DESCRIPTION
**Pull request by dog194**
Now nether wart can be cropped.
※Update for Fortune
Nether wart

### CODE REVIEW

**Comment by CrayonDing** [comment]
+1

[Approved with PullApprove](https://pullapprove.com/iTXTech/Genisys/pull-request/816/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=iTXTech/Genisys)

File: src/pocketmine/item/NetherWart.php [hunk 1]
```diff
@@ -21,8 +21,11 @@
 
 namespace pocketmine\item;
 
+use pocketmine\block\Block;
+
 class NetherWart extends Item {
     public function __construct($meta = 0, $count =1){
+		$this->block = Block::get(Item::NETHER_WART_BLOCK);
```
Comment (ishitatsuyuki) [inline src/pocketmine/item/NetherWart.php]: Please correct indent.

**Comment by Muqsit** [comment]
+1

File: src/pocketmine/item/NetherWart.php (hunk 1 above)
Comment (dog194) [inline src/pocketmine/item/NetherWart.php]: oh. It is an accident.

**Pull request merged**